node/libs/roles/src/proto/validator.proto

```diff
@@ -198,4 +198,9 @@ message AggregateSignature {
 message WeightedValidator {
   optional PublicKey key = 1; // required
   optional uint64 weight = 2; // required
+  optional Signature pop = 3; // required
```

Comment:
use the full name: proof_of_possession.

Reply:
It has to stay optional until we bump the consensus protocol version. Also we need to think through how do we do the rollout, given that we already have external users of the consensus.

Reply:
Changes here: https://github.com/matter-labs/era-consensus/pull/163/commits/1305572a8bc9e3535671be5a6d246133efc1f9f1